pkg/platform/helper.go

```diff
@@ -1007,7 +1007,7 @@ func WaitForVCluster(ctx context.Context, client Client, clusterName, spaceName,
 	}
 
 	now := time.Now()
-	nextMessage := now.Add(waitDuration)
+	nextMessage := now.Add(defaultPollInterval)
```

Comment:
Is it intentional to use this instead of `pollInterval` here?

Reply:
Ah, no.   That function is actually not used anywhere. I'll remove it.